microverse_collins_yemi-main/fcc-test-suite-template-master/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <div class="main">
```

Comment:
Please try using main tag in place of div to make it semantically correct

```suggestion
    <main `class="main">`
```